src/Control/Distributed/Static.hs

```diff
@@ -251,20 +258,62 @@ import Data.Rank1Typeable
   , ANY3
   , ANY4
   , isInstanceOf
+#if __GLASGOW_HASKELL__ >= 710
+  , TypeRep
+#endif
   )
 
+-- Imports necessary to support StaticPtr
+#if __GLASGOW_HASKELL__ >= 710
+import qualified GHC.Prim as GHC (Any)
```

Comment:
This can be imported as `GHC.Exts(Any)` and it would allow to drop the dependency on `ghc-prim`, I think.